.github/ISSUE_TEMPLATE/WG_member_request.yaml

```diff
@@ -0,0 +1,50 @@
+name: Working Group Member Request
+description: Request membership to serve on a PowerShell Working Group
+title: Working Group Member Request
+labels: [WG-NeedsReview, Needs-Triage]
```

Comment:
I wonder if it would help to add all the WG-* labels in case WG's is using that to filter what needs their attention.  Once the right WG is identified, they can remove the other WG labels.